cql3/expr/expression.cc

```diff
@@ -53,6 +53,22 @@ using boost::adaptors::filtered;
 using boost::adaptors::transformed;
 
 
+nested_expression::nested_expression(expression e)
+        : _e(std::make_unique<expression>(std::move(e))) {
+}
+
+nested_expression::nested_expression(const nested_expression& o)
+        : nested_expression(*o._e) {
+}
+
+nested_expression&
+nested_expression::operator=(const nested_expression& o) {
+    if (this != &o) {
```

Comment:
nitpick: it seems to me that this if() is not necessary - the code will be correct (just waste a little time) even without it. But it's not wrong, so ok.